### ISSUE TITLE
Needs some more flexible proxy settings

### ISSUE THREAD

**Issue by the8tre** [eirslett/frontend-maven-plugin#302]
Proxy setting specified in .npmrc are simply ignored and only the maven settings are used.
Adding '--proxy' using the  maven settings simply overrides the .npmrc values.
Some solutions could be:
- Give  .npmrc settings (or grunt or bower ...) priority over maven settings
- Forward all the maven proxy settings to npm, especially the nonProxyHosts. I don't know if npm supports this kind of proxy host exception.

**Comment by eirslett** [comment]
Yes, definitely, .npmrc should have priority (I think). But I think that would be very difficult to implement. (Because the maven plugin passes on proxy settings as --flags, and npm gives flags priority)
